src/app/zap-templates/zcl/data-model/chip/air-quality-cluster.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+  <cluster>
+    <name>Air Quality</name>
+    <domain>Measurement &amp; Sensing</domain>
+    <description>Attributes for reporting air quality classification</description>
+    <code>0x005B</code>
+    <define>AIR_QUALITY_CLUSTER</define>
+    <client tick="false" init="false">true</client>
+    <server tick="false" init="false">true</server>
+
+    <!-- Attributes -->
+    <attribute side="server" code="0x0000" define="AIR_QUALITY" type="AirQualityEnum" min="0" max="5" writable="false" isNullable="false" default="0" optional="false">AirQuality</attribute>
+  </cluster>
+
+  <!-- Feature Map -->
+  <bitmap name="AirQualityFeature" type="BITMAP32">
+    <cluster code="0x005B" />
+    <field name="Fair" mask="0x01" />
+    <field name="Moderate" mask="0x02" />
+    <field name="Very poor" mask="0x04" />
+    <field name="Extremely poor" mask="0x08" />
+  </bitmap>
+
+  <!-- Cluster Data Types -->
+  <enum name="AirQualityEnum" type="ENUM8">
+    <cluster code="0x005B" />
+    <item value="0" name="Unknown" />
+    <item value="1" name="Good" />
+    <item value="2" name="Fair" />
+    <item value="3" name="Poor" />
```

Comment:
Moderate seems missing (at least I have it in my latest spec copy)

Reply:
Yes, that likely explains why this ended up at 5, not 6, for the max.

Reply:
yep, sorry gonna fix this! 👍